### PR TITLE
[RA1 README] Release version

### PR DESCRIPTION
**Pull request by pgoyal01**
Update Release BUndle and chapter staus

### CODE REVIEW

**Review by ildikov — APPROVED** [review]

**Review by walterkozlowski — APPROVED** [review]

**Comment by pgoyal01** [comment]
@karinesevilla @collivier Can your please review? Thanks

**Review by karinesevilla — COMMENTED** [review]
@pgoyal01 Typo: a delimiter is missing for  _6_

**Comment by walterkozlowski** [comment]
@karinesevilla, can you re-review and approve you can? Thanks.

File: doc/ref_arch/openstack/README.md [hunk 1]
```diff
@@ -1,11 +1,10 @@
-[<< Back](https://cntt-n.github.io/CNTT/doc/ref_arch/)
 
 # OpenStack based Reference Architecture
 
 This is OpenStack based Reference Architecture (RA-1)
 
 ## Release Information
-**Bundle: _5_**
+**Bundle: 6_**
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/README.md:7]:
```suggestion
**Bundle: _6_**
```

**Comment by pgoyal01** [comment]
@karinesevilla Thanks - Done

**Review by karinesevilla — APPROVED** [review]

**Comment by pgoyal01** [comment]
@walterkozlowski Can you please merge? Thanks

**Pull request merged**